assets/eip-4910/eip-4910-reference-implementation/LICENSE

```diff
@@ -0,0 +1,223 @@
+                                 Apache License
```

Comment:
Wrong license. Must be CC0.

Reply:
@Pandapip1 Since the reference implementation is optional for an EIP, why does it have to be CC0? @SamWilsn made that point, and there was a discussion amongst editors about that https://github.com/ethereum/EIPs/pull/4910#discussion_r889248615

I am also happy to just link to the external repo and remove the full reference implementation as files from the EIP. The tolling comes with MIT licensing and thus cannot change that. Just lmk what the editorial team decided.

Reply:
@SamWilsn is wrong as of right now. The reference implementation MUST be CC0, as it is part of the EIP.

The reference implementation being optional has nothing to do with this. If you want, feel free to remove it, but as of right now, you must license your reference implementation under CC0. You *may* reference libraries on NPM that are not CC0, ***as long as all the files in the assets folder are CC0***. So you can import OpenZeppelin contracts, but you can't stick the files in your assets folder.

Reply:
@Pandapip1 removed reference implementation and updated the text accordingly. cc @SamWilsn @lightclient